admin/python/lsst/qserv/admin/dataLoader.py

```diff
@@ -82,12 +82,15 @@ class DataLoader(object):
     def __init__(self, configFiles, czarWmgr, workerWmgrMap={}, chunksDir="./loader_chunks",
                  chunkPrefix='chunk', keepChunks=False, skipPart=False, oneTable=False,
                  css=None, cssClear=False, indexDb='qservMeta', tmpDir=None,
-                 emptyChunks=None, deleteTables=False, loggerName=None, cssDb='qservCssData'):
+                 emptyChunks=None, deleteTables=False, loggerName=None, cssDb='qservCssData',
+                 czarWmgrList=[]):
```

Comment:
In Python it is not recommended to use modifiable object (like list or dict) for a default value of parameters. Better way is to use `None` and then test for it in the code.